### DISCUSSION TITLE
Legacy Editor

### DISCUSSION THREAD

**Discussion by TDLBallistic** [AprilSylph/XKit-Rewritten#1147] [Suggestion Box]
So tumblr just went and ruined their own website; every single post you make is now a text post. On dash, on themes, no matter what you do. Old legacy posts made with the old photoset post format still show correctly (art on top, username, caption if there is one, all separate from the image itself).

Glentheme's recently did a fix for blogs, but it's just tricking the blog into making these New Post Format posts LOOK like old post format on the blog. They're still NPF, and still ONLY text posts. Including videos, and gifsets. Text posts are messy and look horrible on most themes, especially ones meant to have multiple gifs or images in a specific order.

If there is any way at all to bring back Legacy Editor, in some way... as tumblr still supports the old posts... I feel like the talented people here would know how to start.

**Comment by marcustyphoon** [comment]
Short version: there is not! If you prefer themes that put the username below the first media element in a post (as there can be as many as you want per post now), you will indeed have to use a theme which makes that adjustment.

**Reply by TDLBallistic** [reply]
Glentheme's fix seems to work with any theme... but is there really no hope of trying to recreate Legacy?
